src/Controls/tests/DeviceTests/Elements/ListView/ListViewTests.cs

```diff
@@ -28,10 +28,65 @@ void SetupBuilder()
 					handlers.AddHandler<ListView, ListViewRenderer>();
 					handlers.AddHandler<VerticalStackLayout, LayoutHandler>();
 					handlers.AddHandler<Label, LabelHandler>();
+					handlers.AddHandler<Entry, EntryHandler>();
 				});
 			});
 		}
 
+
+		[Fact
+#if ANDROID
+			(Skip = "Failing")
```

Comment:
Please create an issue for this and use that as the skip reason (then maybe describe the reason if it is known)

Reply:
Done